collector/cpu_linux.go

```diff
@@ -23,16 +23,22 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/common/log"
 	"github.com/prometheus/procfs"
+	kingpin "gopkg.in/alecthomas/kingpin.v2"
 )
 
 type cpuCollector struct {
 	fs                 procfs.FS
 	cpu                *prometheus.Desc
+	cpuInfo            *prometheus.Desc
 	cpuGuest           *prometheus.Desc
 	cpuCoreThrottle    *prometheus.Desc
 	cpuPackageThrottle *prometheus.Desc
 }
 
+var (
+	enableCPUInfo = kingpin.Flag("collector.cpu.enable-cpu-info-metric", "Enables metric cpu_info").Bool()
```

Comment:
I think this could be simplified to `collector.cpu.info`.

Reply:
Done